### PR TITLE
Create the starting page and add a few links

### PR DESCRIPTION
**Pull request by anamariamilcu**
I created the index starting page and added a few features on it and a few other linked pages.

### CODE REVIEW

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -15,14 +15,31 @@
 /**
  * Adds a random greeting to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js]: These functions are called 'add' but they seem to do replacing instead?

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -15,14 +15,31 @@
...
+    // Add it to the page.
+    const factContainer = document.getElementById("fact-container");
+    factContainer.innerText = chosenFact;
+}
+
+function addPhoto() {
+    const imgElement = document.createElement('img');
+    // Set the url of the image
+    imgElement.src = 'images/friends.jpg';
+    imgElement.width = "600";
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js]: Try to be consistent in your use of single or double quotes.

Style guide suggests single quotes everywhere: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

File: portfolio/src/main/webapp/index.html [hunk 3]
```diff
@@ -2,17 +2,29 @@
...
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hi! I am Ana-Maria and this is my about me page</h1>
+      <p>I am intern for Google's STEP program this year and I'm excited
+          to learn new things. I am 21 years old and I study Computer Science
+          in Bucharest, Romania.</p>
+
+      <p>Here you can see me:</p>
+      <img src="images/me.jpeg" width="600"/>
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/index.html]: Here and elsewhere:

Do not close void elements:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -1,9 +1,41 @@
+body {
+    background-image: url("images/background.png");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-attachment: fixed;
+    background-position: center center;
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/style.css]: nit: Isn't this the same as center?

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (alexcepoi) [reply portfolio/src/main/webapp/script.js]: Ah I see now. This is adding a fact the first time it's clicked and replacing it the second time?

Perhaps a name like addOrReplaceFact() would make more sense? You can also leave it as is, if you prefer.

**Review by alexcepoi — COMMENTED** [review]
Left some minor comments.

Please address and let me have a final look.

File: portfolio/src/main/webapp/style.css (hunk 4 above)
Comment (anamariamilcu) [reply portfolio/src/main/webapp/style.css]: I am not sure what do you mean by this?

Comment (alexcepoi) [reply portfolio/src/main/webapp/style.css]: Just that "background-position: center;" should work equally well?

"background-position: center center;" feels a bit redundant.

File: README.md [hunk 5]
```diff
@@ -1,13 +1,3 @@
...
-
-To get started:
-
-- Login to [Google Cloud Shell](https://ssh.cloud.google.com/cloudshell/editor)
-- Clone this repo: `cd; git clone https://github.com/googleinterns/step.git`
-- Open the GitHub setup walkthrough: `teachme ~/step/walkthroughs/week-1-setup/github-setup-walkthrough.md`
-  - If the tutorial panel does not open or display the walkthrough contents, try refreshing the page.
-
-Then follow the on-screen instructions to set up your repo.
+This repo contains Ana-Maria's portofolio and STEP projects.
```
Comment (alexcepoi) [inline README.md:3]: nit: portfolio*

File: portfolio/src/main/webapp/index.html [hunk 6]
```diff
@@ -2,17 +2,29 @@
...
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+      <h1>Hi! I am Ana-Maria and this is my about me page</h1>
+      <p>I am intern for Google's STEP program this year and I'm excited
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/index.html:11]: ultranit: I'm an* intern or I am interning

File: portfolio/src/main/webapp/script.js [hunk 7]
```diff
@@ -15,14 +15,32 @@
 /**
  * Adds a random greeting to the page.
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js:16]: Comment talks about a greeting, yet the function adds a fact (or replaces it if one already exists).

File: portfolio/src/main/webapp/script.js [hunk 8]
```diff
@@ -15,14 +15,32 @@
...
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+function addAndReplaceRandomFact() {
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js:19]: nit: addOrReplaceRandomFact? (I think you're either adding a fact if this is the first press or replacing it on subsequent presses).

Same for addPhoto.

File: portfolio/src/main/webapp/script.js [hunk 9]
```diff
@@ -15,14 +15,32 @@
...
+  const chosenFact = facts[Math.floor(Math.random() * facts.length)];
 
   // Add it to the page.
-  const greetingContainer = document.getElementById('greeting-container');
-  greetingContainer.innerText = greeting;
+  const factContainer = document.getElementById('fact-container');
+  factContainer.innerText = chosenFact;
+}
+
+function addPhoto() {
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js:36]: Why not add a function comment? Functions should generally have a short description.
Moreso that the previous very similar function has a comment.

https://google.github.io/styleguide/jsguide.html#jsdoc-method-and-function-comments

File: portfolio/src/main/webapp/script.js [hunk 10]
```diff
@@ -15,14 +15,32 @@
...
+  const imgElement = document.createElement('img');
+  // Set the url of the image
```
Comment (alexcepoi) [inline portfolio/src/main/webapp/script.js:38]: I took a screenshot on some guidance on what constitutes good and bad comments:
https://screenshot.googleplex.com/rfSQPbHU5UA

Also pointing to some guidance in the C++ styleguide (this is relatively unrelated to C++):
https://google.github.io/styleguide/cppguide.html#Implementation_Comment_Donts
"Do not state the obvious. In particular, don't literally describe what code does, unless the behavior is nonobvious to a reader who understands C++ well. Instead, provide higher level comments that describe why the code does what it does, or make the code self describing."

In this particular case I could probably argue that a comment like "Set the url of the image" is pretty self explanatory, while a comment like "Remove the previous image." is genuinely useful as it's not totally clear why is innerHTML set to ''.

This is totally not a big deal, though might be useful to keep in mind whenever you're thinking about implementation comments.

**Review by alexcepoi — APPROVED** [review]
Looking good!

Congrats on your first PR!

File: portfolio/src/main/webapp/script.js (hunk 10 above)
Comment (anamariamilcu) [reply portfolio/src/main/webapp/script.js:38]: Alright, thank you, I will try to keep this in mind from no on.

**Pull request merged**